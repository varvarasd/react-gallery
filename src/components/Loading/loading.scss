.loading {
    font-size: 30px;
    font-weight: 500;
}