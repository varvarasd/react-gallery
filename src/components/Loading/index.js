import React from 'react';
import './loading.scss';

const Loading = ({text}) => <h1 className="loading">Loading {text}...</h1>;

export default Loading;