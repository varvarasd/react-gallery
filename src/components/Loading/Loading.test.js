import React from 'react';
import { render } from '@testing-library/react';
import Loading from '.';

describe('Loading', () => {
    it('renders', () => {
        const { container } = render(<Loading />);
        expect(container.firstChild).toMatchSnapshot();
    });

    it('renders children', () => {
        const { getByText } = render(<Loading text='some text' />);
        expect(getByText(/some text/)).toBeInTheDocument();
    });
});