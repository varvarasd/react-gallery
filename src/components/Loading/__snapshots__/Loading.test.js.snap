// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders 1`] = `
<h1
  class="loading"
>
  Loading 
  ...
</h1>
`;
